test/truffle/docker/oraclelinux/Dockerfile

```diff
@@ -21,9 +24,15 @@ USER test
 ENV GRAALVM_VERSION=0.24
 ADD graalvm-$GRAALVM_VERSION-linux-amd64-jdk8.tar.gz /test
 ENV PATH=/test/graalvm-$GRAALVM_VERSION/bin:$PATH
+ENV RUBY_HOME=/test/graalvm-$GRAALVM_VERSION/language/ruby
+
+# Build the AOT image
+RUN aot-image -H:Name=aot-ruby --ruby
+ENV PATH=/test:$PATH
 
 # Straight away, just run the ruby executable as the most basic test
 RUN ruby --version
+RUN aot-ruby -Xhome=$RUBY_HOME --version
```

Comment:
I made it that location under the assumption you'd `cd` into the root and run `./bin/aot-image`. It's arbitrary, but since we can't put it in `bin` without renaming it seemed as good a place as any. And it's what the test suite uses.

The AOT image does know its absolute path, if that's helpful. That's what https://github.com/graalvm/truffleruby/blob/59a2497b24e6cb95082cd273fa019a290be88438/src/main/java/org/truffleruby/RubyContext.java#L388 is doing.

But, yes, if you use relative paths and then try to move the binary somewhere else, you're going to need to set `-Xhome`. I don't see a way around that short of building everything into the image.